code/__DEFINES/vv.dm

```diff
@@ -62,8 +66,7 @@
 #define VV_HK_TARGET "target"
 ///name or index of var for 1 variable targetting hrefs.
 #define VV_HK_VARNAME "targetvar"
-/// to view an appearance virtual object
-#define VV_HK_VIEW_APPEARANCE "vv_appearance"
+// VV_HK_VIEW_APPEARANCE is handled on vv natively now
```

Comment:
just get rid of the comment, anyone touching vv here knows what they're doing / shouldn't need a comment to know why